.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "${workspaceFolder}/venv/bin/python"
+}
```

Comment:
EOF missing here

Reply:
Thank you! I will add EOF where required. I have one question though, is it possible to add changes to the same pull request or I am required to make a new pull request.